xcode/cppjansson/cppjansson.xcodeproj/project.pbxproj

```diff
@@ -158,9 +158,7 @@
 			baseConfigurationReference = 71407E71294CB35000FBD9A4 /* common-debug.xcconfig */;
 			buildSettings = {
 				HEADER_SEARCH_PATHS = (
-                    "$(PROJECT_DIR)/../../",
-                    "$(PROJECT_DIR)/../../3party/jansson",
-                    "$(PROJECT_DIR)/../../3party/jansson/jansson/src",
+					"$(inherited)",
```

Comment:
Revert this file at all.

Reply:
Hi @vng,
Thank you for your kind review.
In this case, such a change or an equivalent appears to be needed, because otherwise the `cppjansson` project doesn't inherit, the settings from `common.xcconfig`, and the build fails:

````
/Build/Intermediates.noindex/cppjansson.build/Debug-iphoneos/cppjansson.build/Objects-normal/arm64/cppjansson.o
In file included from /Users/runner/work/organicmaps/organicmaps/cppjansson/cppjansson.cpp:1:
In file included from /Users/runner/work/organicmaps/organicmaps/cppjansson/cppjansson.hpp:6:
/Users/runner/work/organicmaps/organicmaps/base/string_utils.hpp:21:10: fatal error: 'utf8cpp/utf8/unchecked.h' file not found
#include <utf8cpp/utf8/unchecked.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
````